src/stories/Card/Card.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import {
+  StyledDiscription,
+  StyledImage,
+  StyledTitle,
+  StyledBadge,
+  StyledDissmiss,
+  StyledOverLay,
+  StyledContainer,
+} from "./Card.style";
+import Button from "../Button/Button";
+
+const Cards = ({
+  title,
+  discription,
+  alt,
+  imageSrc,
+  badge,
+  dismiss,
```

Comment:
@PRABgit01 give proper name to `dismiss`

Reply:
mam this name was given by sir on addon